### PR TITLE
MCS-1925 - [Webenabled] Ignoring non-action keys

### PR DESCRIPTION
**Pull request by rartiss55**
Original ticket was to not submit a pass action for function keys -- decided to simply ignore all non-action keys. Also addressed a small bug where you could submit an action before actually choosing a scene. 

Note that this is branched off of MCS-1816-styling.

### CODE REVIEW

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]
That does the trick for me.

Quick observation (probably a separate ticket): When I hit spacebar, my browser automatically scrolls down, which is unfortunate.

File: webenabled/templates/mcs_page.html [hunk 1]
```diff
@@ -360,7 +360,8 @@ <h1>Machine Common Sense</h1>
     // Handle clicking on one of the scene file names
     var sceneSelect = document.getElementById("scenes-dropdown");
     let actionKeysWithParams = ['1', '3', '4', '5', '6', '7', '8', '9', 'm', 'M', 't', 'T']
-    let arrowKeys = ["ArrowDown", "ArrowUp", "ArrowLeft", "ArrowRight"]
+    let allOtherActionKeys = ['2', 'w', 'W', 'a', 'A', 's', 'D', 'h', 'H', 'i', 'I', 'j',
```
Comment (phile-caci) [inline webenabled/templates/mcs_page.html:363]: Think we're missing lower case 'd'.

Comment (ThomasSchellenbergNextCentury) [reply webenabled/templates/mcs_page.html:363]: I think we could just use uppercase letters and say `!allOtherActionKeys.includes(e.key.toUpperCase())`

**Comment by rartiss55** [comment]
> That does the trick for me.
> 
> Quick observation (probably a separate ticket): When I hit spacebar, my browser automatically scrolls down, which is unfortunate.

Good catch, the spacebar shouldn't scroll anymore.

File: webenabled/templates/mcs_page.html (hunk 1 above)
Comment (rartiss55) [reply webenabled/templates/mcs_page.html]: Thanks for catching this -- fixed.

**Pull request merged**